website/docs/howto.md

```diff
@@ -136,3 +136,18 @@ You can list all constraints in a cluster with the following command:
 ```sh
 kubectl get constraints
 ```
+
+### Input Review
+
+The `input.review` object stores the admission request under evaulation. It has the following fields:
+- `dryRun`: Describes if the request was invoked by `kubectl --dry-run`. This is not available during audit.
+- `kind`: The resource `kind`, `group`, `version` of the request object under evaulation.
+- `name`: The name of the request object under evaulation. It may be empty if the deployment expects the API server to generate a name for the requsted resource.
```

Comment:
```suggestion
- `name`: The name of the request object under evaluation. It may be empty if the deployment expects the API server to generate a name for the requested resource.
```